onnxruntime/core/providers/cann/cann_execution_provider.cc

```diff
@@ -746,4 +747,14 @@ void CANNExecutionProvider::RegisterAllocator(AllocatorManager& allocator_manage
   }
 }
 
+common::Status CANNExecutionProvider::CheckSessionOptionsAndProcess(const SessionOptions& so) {
+  if (so.execution_mode != ExecutionMode::ORT_SEQUENTIAL) {
+    return ORT_MAKE_STATUS(ONNXRUNTIME, FAIL,
+                           "Parallel execution mode does not support the CANN Execution Provider. "
```

Comment:
This line is a bit misleading. May be we can say that parallel execution mode is incompatible with this execution provider?

Reply:
Sounds good.